moveit_planners/chomp/chomp_interface/CMakeLists.txt

```diff
@@ -55,11 +56,11 @@ install(TARGETS ${PROJECT_NAME} chomp_planner_plugin
 )
 
 if(CATKIN_ENABLE_TESTING)
-  find_package(catkin REQUIRED COMPONENTS rostest roscpp moveit_ros_planning_interface)
+  # additional packages needed for testing
+  find_package(catkin REQUIRED COMPONENTS rostest moveit_ros_planning_interface)
```

Comment:
shouldn't this be
```
find_package(rostest REQUIRED)
find_package(moveit_ros_planning_interface REQUIRED)
```
?
We already `find_package` catkin further above and one shouldn't lookup the same package twice with different options.

Reply:
I just moved the corresponding `find_package` calls as they were. Changing this, requires to explicitly refer to package-specific variables instead of using the convenient `catkin_*` vars.